project-api/Dockerfile.template

```diff
@@ -1,11 +1,17 @@
-
-   
-FROM balenalib/%%BALENA_MACHINE_NAME%%-node:16
+FROM balenalib/%%BALENA_MACHINE_NAME%%-alpine-node:16
 
 WORKDIR /usr/src/app
+
+# install build packages
+RUN apk --no-cache add python2 python3 py3-pip make g++
```

Comment:
These are necessary in order to build binaries for the sqlite npm dependency. Just followed build error output and found which alpine packages provided the necessary utilities.